server/app/views/applicant/ApplicantProgramBlockEditView.java

```diff
@@ -98,6 +99,20 @@ private ContainerTag<?> renderBlockWithSubmitForm(Params params) {
     if (params.block().isFileUpload()) {
       return fileUploadStrategy.renderFileUploadBlock(params, applicantQuestionRendererFactory);
     }
+
+    FormTag form = form();
+
+    if (params.block().hasErrors()
+        && params.errorDisplayMode()
+            == ApplicantQuestionRendererParams.ErrorDisplayMode.DISPLAY_ERRORS) {
```

Comment:
suggestion (blocking): use `.equals` for comparing enums
```suggestion
        && ApplicantQuestionRendererParams.ErrorDisplayMode.DISPLAY_ERRORS.equals(params.errorDisplayMode())) {
```